src/js/components/homepage/features/PaneFeature.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * PaneFeature.jsx
+ * Created by David Trinh 4/19/18
+ */
+
+import React from 'react';
+
+const PaneFeature = () => (
+    <div className="feature-pane">
+        <div className="feature-pane__wrapper">
+            <h2 className="feature-pane__title"><span>FEATURED CONTENT</span></h2>
```

Comment:
The title should be the content of the `h2` tag. Instead of styling the `h2` tag to be the grey border line and then styling the inner text using `span`, I would recommend making the text an `h2` (no span) and then wrapping the `h2` inside a `div` that provides the border styling.

Additionally, please write the title in normal sentence case ("Featured Content") and use `text-transform` to display it as upper case. This allows users who have trouble reading all caps text to override with a custom local stylesheet.